doc/TESTING.md

```diff
@@ -39,11 +39,18 @@ You have some options to avoid running them as root:
    run `ctest` without tests that have the label `kdbtests`:
    `ctest --output-on-failure -LE kdbtests`
    (which is also what `make run_nokdbtests` does)
-2. To give your user the permissions to the relevant paths, i.e. (once as root):
+2. To give your user the permissions to the relevant paths execute the lines
+   below once as root.
+   First load the required information and make sure that all paths are correct:
    ```
    kdb mount-info
-   chown -R `whoami` `kdb get system/info/constants/cmake/CMAKE_INSTALL_PREFIX`/`kdb get system/info/constants/cmake/KDB_DB_SPEC`
-   chown -R `whoami` `kdb get system/info/constants/cmake/KDB_DB_SYSTEM`
+   echo `kdb get system/info/elektra/constants/cmake/CMAKE_INSTALL_PREFIX`/`kdb get system/info/elektra/constants/cmake/KDB_DB_SPEC`
+   echo `kdb get system/info/elektra/constants/cmake/KDB_DB_SYSTEM`
+   ```
+   Then change the permissions:
+   ```
+   chown -R `whoami` `kdb get system/info/elektra/constants/cmake/CMAKE_INSTALL_PREFIX`/`kdb get system/info/constants/cmake/KDB_DB_SPEC`
```

Comment:
What about using `sget` and defaulting to `.` then it won't harm the root.

We should also explicitly warn that changing the permissions can be harmful on wrong paths (backup needed).